packages/mui-material/src/Select/Select.test.js

```diff
@@ -292,6 +292,16 @@ describe('<Select />', () => {
     });
   });
 
+  describe('prop: defaultOpen', () => {
+    it('should be open on mount', () => {
+      render(<Select defaultOpen value="" />);
+      expect(document.querySelector('div[role="button"]')).to.have.attribute(
+        'aria-expanded',
+        'true',
+      );
```

Comment:
Use available helper functions where possible:

```suggestion
      const { getByRole } = render(<Select defaultOpen value="" />);
      expect(getByRole('button')).to.have.attribute(
        'aria-expanded',
        'true',
      );
```

Reply:
Updated :)